[2022-02-03 20:45:43,329] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_3.download_dataset_task scheduled__2019-01-02T06:00:00+00:00 [queued]>
[2022-02-03 20:45:43,370] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_3.download_dataset_task scheduled__2019-01-02T06:00:00+00:00 [queued]>
[2022-02-03 20:45:43,372] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 20:45:43,373] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-03 20:45:43,374] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 20:45:43,414] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-01-02 06:00:00+00:00
[2022-02-03 20:45:43,425] {standard_task_runner.py:52} INFO - Started process 5950 to run task
[2022-02-03 20:45:43,433] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag_3', 'download_dataset_task', 'scheduled__2019-01-02T06:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag_2.py', '--cfg-path', '/tmp/tmpoqa5ss33', '--error-file', '/tmp/tmp8xe2zecw']
[2022-02-03 20:45:43,436] {standard_task_runner.py:77} INFO - Job 31: Subtask download_dataset_task
[2022-02-03 20:45:43,557] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag_3.download_dataset_task scheduled__2019-01-02T06:00:00+00:00 [running]> on host 985445ce9491
[2022-02-03 20:45:43,699] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag_3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-02T06:00:00+00:00
[2022-02-03 20:45:43,705] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-03 20:45:43,707] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'pwd']
[2022-02-03 20:45:43,725] {subprocess.py:85} INFO - Output:
[2022-02-03 20:45:43,768] {subprocess.py:89} INFO - /tmp/***tmp8918f7jh
[2022-02-03 20:45:43,772] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-03 20:45:43,855] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag_3, task_id=download_dataset_task, execution_date=20190102T060000, start_date=20220203T204543, end_date=20220203T204543
[2022-02-03 20:45:43,930] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 20:45:44,013] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
